onnxruntime/core/framework/execution_frame.cc

```diff
@@ -222,8 +222,8 @@ ExecutionFrame::ExecutionFrame(const std::vector<int>& feed_mlvalue_idxs, const
                              : nullptr;
           buffers_[mem_patterns_->locations[i]] = BufferUniquePtr(buffer, alloc);
 
-          // comment out following line to see the size of activation
-          // printf("Allocated memory for activations, size: %zu\n", mem_patterns_->patterns[i].PeakSize());
+          // log size of activation
+          VLOGS(session_state_.Logger(), 1) << "Allocated memory for activations, size: " << mem_patterns_->patterns[i].PeakSize();
```

Comment:
Please keep this commented out for now... 
Otherwise, this will flood the log...

Reply:
ok

Reply:
VLOG is already for verbose debug logging, so i think flooding is ok. i kind of agree with the original comment.